README.md

```diff
@@ -73,6 +98,12 @@ from the `conda-forge` channel, using one of the following two commands:
     $ conda install -c conda-forge jupyter-ai  # or,
     $ conda install conda-forge::jupyter-ai
 
+In addition, install third-party `langchain-<provider>` packages, to use some model providers. For example, if you are using OpenAI models:
```

Comment:
The wording in this sentence is a bit confusing. 

````suggestion
Most model providers in Jupyter AI require a specific dependency to be installed before they are available for use. These provider dependencies are not installed by default when installing `jupyter-ai` from Conda Forge, and should be installed separately as-needed.

For example, to install Jupyter AI with support for OpenAI models, run:

```
conda install conda-forge::jupyter-ai conda-forge::langchain-openai
```

For more information on model providers and which dependencies they require, see [the model provider table](https://jupyter-ai.readthedocs.io/en/latest/users/index.html#model-providers).
````

Reply:
Please also make this change in the user docs.